images/runtime/dotnetcore/generateDockerfiles.sh

```diff
@@ -25,12 +25,15 @@ declare -r DOTNET_VERSION_PLACEHOLDER="%DOTNET_VERSION%"
 # Please make sure that any changes to debian flavors supported here are also reflected in build/constants.yaml
 declare -r NETCORE_BUSTER_VERSION_ARRAY=($NET_CORE_APP_30 $NET_CORE_APP_31 $NET_CORE_APP_50 $NET_CORE_APP_60 $NET_CORE_APP_70)
 declare -r NETCORE_BULLSEYE_VERSION_ARRAY=()
```

Comment:
Just for my understanding, do you know how we are able to already provide a jamstack image to our partners for Bullseye without generating a Dockerfile here for Bullseye? Is it that jamstack support is handled somewhere else?